core/chains/evm/client/errors.go

```diff
@@ -296,14 +296,18 @@ var sei = ClientErrors{
 	Fatal: regexp.MustCompile(`(: |^)'*out of gas|insufficient fee|Tx too large. Max size is \d+, but got \d+|: insufficient funds`),
 }
 
+var monad = ClientErrors{
+	TerminallyUnderpriced: regexp.MustCompile("Gas limit too low"),
```

Comment:
Are they really returning an error message that is this inaccurate?

Reply:
Yup :/ From our testing:
```
 Transaction should fail with error intrinsic gas too low, got unexpected error
       - Cause: Gas limit too low
 ```

Reply:
That looks like a normal gas limit error. What makes you think that it is related to gas price?

Reply:
The test this is from specifically sets a too low of a gas limit. I can actually move this over to a fatal error to better classify it (falling in line with geth intrinsic gas too low error)

Reply:
updated the classification to fatal 👍